src/protocols/ssh/ssh.c

```diff
@@ -318,11 +323,18 @@ void* ssh_client_thread(void* data) {
 
     /* While data available, write to terminal */
     int bytes_read = 0;
+    int timeout = 0;
```

Comment:
The scope of `timeout` can be tightened - there's no need to declare it outside the only block which uses its value (the `if` block within the loop).